src/main/java/ladder/domain/RandomLineStrategy.java

```diff
@@ -0,0 +1,33 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RandomLineStrategy implements LineStrategy {
+    Random random = new Random();
+
+    @Override
+    public List<Boolean> points(int countOfPerson) {
+        List<Boolean> points = new ArrayList<>();
+
+        boolean previousLine = false;
+
+        for (int i = 0; i < countOfPerson; i++) {
+            boolean point = calculatePoint(previousLine, countOfPerson, i);
+            points.add(point);
+            previousLine = point;
+        }
+
+        return points;
+    }
+
+    private boolean calculatePoint(boolean previousLine, int countOfPerson, int i) {
+        boolean line = random.nextBoolean();
+        if (line && !previousLine && i != (countOfPerson - 1)) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
```suggestion
    private boolean calculatePoint(boolean previousLine, int countOfPerson, int i) {
        boolean line = random.nextBoolean();
        return line && !previousLine && i != (countOfPerson - 1);
    }
```
다음과 같이 해줄 수 있을 것 같아요! 😄 

또한 i 라는 네이밍으로는 어떤 값을 의미하는지 파악이 힘든데요
이름 만으로 어떠한 값을 의미하는지 알 수 있도록 네이밍을 수정해보면 좋을 것 같아요! 😄